hedera-node/hedera-admin-service-impl/src/main/java/com/hedera/node/app/service/admin/impl/config/AdminServiceConfig.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.admin.impl.config;
+
+import com.hedera.node.app.spi.config.PropertyNames;
+import com.swirlds.config.api.ConfigProperty;
+
+public record AdminServiceConfig(@ConfigProperty(PropertyNames.UPGRADE_ARTIFACTS_PATH) String upgradeArtifactsPath) {}
```

Comment:
Can you add documentation for the property? See https://github.com/hashgraph/hedera-services/blob/develop/platform-sdk/swirlds-common/src/main/java/com/swirlds/common/config/StateConfig.java for example. Does it makes sense to define a default?

Reply:
Fixed